packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/parts/EnsembleMappingRow.java

```diff
@@ -0,0 +1,21 @@
+package software.uncharted.terarium.hmiserver.models.simulationservice.parts;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import java.util.Map;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+
+@Data
+@Accessors(chain = true)
+@TSModel
+public class EnsembleMappingRow {
```

Comment:
Can you update this to include the name Calibrate?
The simulate version of this mapping will not include a dataset_mapping so it will be its own unique type.

Reply:
will do!